metadata/index.html

```diff
@@ -1453,11 +1467,12 @@
           What should be the mapping of an empty cell? 
         </p>
         <ol>
-          <li>unless the <code>datatype</code> is <code>string</code> or <code>anySimpleType</code> or <code>any</code>, strip leading and trailing whitespace from the value</li>
+          <li>unless the <a href="#cell-valueUrl"><code>valueUrl</code></a> is defined on the cell, <code>datatype</code> is <code>string</code> or <code>anySimpleType</code> or <code>any</code>, strip leading and trailing whitespace from the value</li>
```

Comment:
I'm not sure about this: if the value is a date or a number, you'd want to strip whitespace from around it before using that value to create the URL value from the `valueUrl` template, wouldn't you? In the usual case, the `datatype` will be the default (`string`) and whitespace won't be stripped.

But if keeping as is:

s/defined on the cell,/defined on the cell, or/